pycbc/inference/sampler/multinest.py

```diff
@@ -396,4 +397,9 @@ def from_config(cls, cp, model, nprocesses=1, use_mpi=False):
                            v is not None}
         obj = cls(model, nlivepoints, constraints=constraints,
                   **optional_kwargs)
+        obj.setup_output(obj, output_file)
+        if not obj.new_checkpoint:
+            obj.resume_from_checkpoint(cp)
+        else:
+            init_prior = initial_dist_from_config(cp, obj.variable_params)
```

Comment:
In these cases where the sampler doesn't have a `set_p0` method, is there a reason this is still making a call to `initial_dist_from_config`?